### ISSUE TITLE
References to be considered for the senior members of the club

### ISSUE THREAD

**Issue by Vedaant-Rajoo** [utkarshsingh99/ClubComing#16]
While conducting interviews for the intake of students to the different clubs, the references of the senior members of the club need to be considered, many senior members do not attend the interview and their input is crucial for the selection process.

**Comment by utkarshsingh99** [comment]
Can't we comment the reference for a particular candidate?

**Comment by Vedaant-Rajoo** [comment]
Yeah we can, but that way won't the recognition of the senior member be difficult.
I'm thinking  of something on the grounds of linkedin's feature of endorsing.
So we know exactly who is supporting whom.
What say?
